components/BlurryPoster/BlurryPoster.tsx

```diff
@@ -187,7 +187,7 @@ export const BlurryPoster = ({
                   className={styles.image}
                 />
                 <img
-                  src={getProcessedImage(imageSrc, type === 'hero' ? '1000x1000' : '1000x500', imageFocus)}
+                  src={getProcessedImage(imageSrc, '1000x1000', imageFocus)}
```

Comment:
Use Square image for mobile BlurryPoster instead of 2x1 image so it works better for images like this
![Screenshot 2023-11-29 at 4 09 50 PM](https://github.com/SU-SWS/ood-giving-campaign/assets/42749717/06506df0-1227-4c34-bf35-6837d80acd56)